pkg/provisioning/config/parser.go

```diff
@@ -39,8 +39,10 @@ type Connector struct {
 }
 
 type Processor struct {
```

Comment:
Thinking out loud - should we add the same changes in `yaml/v1`? It seems that for completeness we would need to do that.

Reply:
I was thinking about that too... I believe that we should be encouraging users to use v2 as much as possible,